CHANGELOG.rst

```diff
@@ -577,7 +578,7 @@ v0.10.0
   See https://github.com/xonsh/xonsh/pull/4283 and linked issues.
 * Wrap selection with quote/parens when ``$XONSH_AUTOPAIR=True``.
 * Now xonsh will work with Python 3.10. (Match statement is not supported).
-* In addition to reading single rc files at startup (``/etc/xonshrc``, ``~/.config/xonsh/rc.xsh``),
+* In addition to reading single rc files at startup (``/etc/xonsh/xonshrc``, ``~/.config/xonsh/rc.xsh``),
```

Comment:
This entry belongs to the additions list of v0.10.0. Was it `/etc/xonsh/xonshrc` already then? As far as I could determine (see https://github.com/NixOS/nixpkgs/pull/204252#issuecomment-1336156734), the change occurred with 0.12.0 and that [in 0.10.0](https://github.com/xonsh/xonsh/blob/0.10.0/xonsh/environ.py#L591) the old path `/etc/xonshrc` was still in use.